admin/requirements.in

```diff
@@ -1,2 +1,3 @@
 prompt_toolkit
 pyyaml
+cryptography==2.7
```

Comment:
Can you squash the commits related to this version bump? It'd be great have the code diff for the key generation (which requires the version bump) in the same commit as the version bump. It's also not clear which parts of cryptography needed to be updated—presumably the x25519 calls? Documenting that in the commit message will be hugely helpful to future maintainers.

Reply:
Yes, I will do that.

Reply:
When resolving the reqs commits, consider also appending the cryptography bump to the dev env vars: if the script is executed by Ansible (see rationale in https://github.com/freedomofpress/securedrop/pull/4652#discussion_r312283362), then it must have a recent version of cryptography available (or else we use bash). More detail on _why_ the version bump to crytpography is required here would be helpful in determining the best way to generate the keypairs!